config.env.py

```diff
@@ -38,5 +38,10 @@
 MAIL_PASSWORD = environ.get("PACKET_MAIL_PASSWORD", None)
 MAIL_USE_TLS = strtobool(environ.get("PACKET_MAIL_TLS", 'True'))
 
+# OneSignal Config
+ONESIGNAL_USER_AUTH_KEY = environ.get("PACKET_ONESIGNAL_USER_AUTH_KEY", None)
+ONESIGNAL_APP_AUTH_KEY = environ.get("PACKET_ONESIGNAL_APP_AUTH_KEY", None)
+ONESIGNAL_APP_ID = environ.get("PACKET_ONESIGNAL_APP_ID", "6eff123a-0852-4027-804e-723044756f00")
```

Comment:
This is a test ID that only works on localhost:6969 for testing

Reply:
You may want to note that in a line comment then. It'd help us track what is needed for local testing.